packages/react-reconciler/src/ReactFiber.new.js

```diff
@@ -496,6 +496,11 @@ export function createFiberFromTypeAndProps(
         return createFiberFromOffscreen(pendingProps, mode, lanes, key);
       case REACT_LEGACY_HIDDEN_TYPE:
         return createFiberFromLegacyHidden(pendingProps, mode, lanes, key);
+      case REACT_SCOPE_TYPE:
+        if (enableScopeAPI) {
+          return createFiberFromScope(type, pendingProps, mode, lanes, key);
+        }
+        break;
```

Comment:
This `break` doesn't look right to me. If the feature flag is off, I think we want it to full through to the default case where it would throw with a nice message. Is that not right?

Reply:
It would still fall through the the error case, which doesn't happen in `default` but rather by the fact we aren't doing a `return`.

Reply:
Ah yes, that's right.